.azure-pipelines/client.yml

```diff
@@ -83,18 +84,87 @@ jobs:
         condition: and(succeededOrFailed(), ne(variables['Build.Reason'], 'PullRequest'))
         displayName: 'Component Detection'
 
-  - template: client.test.yml
-    parameters:
-      name: Linux
-      vmImage: 'ubuntu-16.04'
+  - job: 'Test'
+
+    strategy:
+      matrix:
+        Linux_Node8:
+          os.name: 'Linux'
+          os.vmImage: 'ubuntu-16.04'
+          node.version: '8.x'
+        Linux_Node10:
+          os.name: 'Linux'
+          os.vmImage: 'ubuntu-16.04'
+          node.version: '10.x'
+        Linux_Node11:
+          os.name: 'Linux'
+          os.vmImage: 'ubuntu-16.04'
+          node.version: '11.x'
+        macOS_Node8:
+          os.name: 'macOS'
+          os.vmImage: 'macOS-10.13'
+          node.version: '8.x'
+        macOS_Node10:
+          os.name: 'macOS'
+          os.vmImage: 'macOS-10.13'
+          node.version: '10.x'
+        macOS_Node11:
+          os.name: 'macOS'
+          os.vmImage: 'macOS-10.13'
+          node.version: '11.x'       
+        Windows_Node8:
+          os.name: 'Windows'
+          os.vmImage: 'vs2017-win2016'
+          node.version: '8.x'
+        Windows_Node10:
+          os.name: 'Windows'
+          os.vmImage: 'vs2017-win2016'
+          node.version: '10.x'
+        Windows_Node11:
+          os.name: 'Windows'
+          os.vmImage: 'vs2017-win2016'
+          node.version: '11.x'
+
+    pool:
+      vmImage: '$(os.vmImage)'
+
+    steps:
+      - task: NodeTool@0
+        inputs:
+          versionSpec: '$(node.version)'
+        displayName: 'Install Node.js $(node.version)'
+
+      # Rather than running "npm install" in the repo root, reduce build time by only installing the "npm-run-all" package,
+      # which is the minimum required to run the composite script commands.
+      # 
+      # Also, "npm install" in the repo root fails when using Node11 on Windows or Linux
+      # (https://github.com/Azure/azure-sdk-for-js/issues/845)
+      - task: Npm@1
+        inputs:
+          command: 'custom'
+          customCommand: 'install npm-run-all'
+        displayName: 'npm install npm-run-all'
+
+      - task: Npm@1
+        inputs:
+          command: 'custom'
+          customCommand: 'run install-client'
+        displayName: 'npm run install-client'
 
-  - template: client.test.yml
-    parameters:
-      name: macOS
-      vmImage: 'macOS-10.13'
+      - task: Npm@1
+        inputs:
+          command: 'custom'
+          customCommand: 'run test-client -- -- -- --reporter mocha-junit-reporter'
+        displayName: 'npm run test-client'
+
+      - task: PublishTestResults@2
+        inputs:
+          testResultsFiles: '**/test-results.xml'
+          testRunTitle: '$(os.name) Node $(node.version)'
+        condition: succeededOrFailed()
+        displayName: 'Publish test results'
+    
+ 
+ 
```

Comment:
Remove extra whitespace.  Line 165 should end with `\r\n`, but then no blank lines after it.